modules/p2/ding_list/ding_list.module

```diff
@@ -435,6 +435,11 @@ function ding_list_ctools_plugin_directory($owner, $plugin_type) {
  * loans list.
  */
 function ding_list_user_login(&$edit, $account) {
+  // Don't update users when openlist isn't available.
+  if (ding_provider_implements('openlist', 'is_available') && !ding_provider_invoke('openlist', 'is_available')) {
+    return;
+  }
```

Comment:
User login is called during installation, I assume perhaps from drush to aquire root permissions for all further actions.

We could filter for only provider role, but I believe that for many purposes the lists should work for all users including uid=1.

We havn't found a proper way to determine if the PHP session is part of an installation or a normal production call. If we have such a bit we should use that instead. Any ideas?